Tutorial 4

VicAzv opened the pull request:
Criação do Joint Call Genotypes com o uso de scatter

IsadoraRFMuniz committed:
Tutorial 4

desouzareis commented on joint-calling-genotypes.wdl:
```diff
@@ -0,0 +1,117 @@
+            refDict = refDict, 
+            sampleName = "CEUtrio", 
+            GVCFs = HaplotypeCallerERC.GVCF
+        
```
Uma linha vazia a mais

desouzareis commented on joint-calling-genotypes.wdl:
```diff
@@ -0,0 +1,117 @@
+
+    output {
+        File GVCF = "~{sampleName}_rawLikelihoods.g.vcf"
+    }
```
Poderia ter o parameter_meta explicando cada um dos inputs e outputs de cada task.

desouzareis reviewed (approved):
Muito bom! Parabéns! Deixei apenas algumas sugestões de melhorias.

VicAzv committed:
Correção de espaçamento e adição do parameter_meta

Foi corrigido o espaçamento na linha 34 e adicionado o bloco "parameter_meta" em todas as tasks
